src/integration-test/tests/citizen/defence/pages/statement-of-means/carer.ts

```diff
@@ -0,0 +1,25 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  yes: 'input[id=optionyes]',
+  no: 'input[id=optionno]'
+}
+
+const buttons = {
+  submit: 'input[type=submit]'
+}
+
+export class CarerPage {
+
+  selectYesOption (): void {
+    I.checkOption(fields.yes)
```

Comment:
So` I.checkOption(fields.IsCarer)` more meaningful if radiobutton field is renamed